.github/workflows/_check-excluded-packages.yml

```diff
@@ -43,18 +43,19 @@ jobs:
 
       - name: Check excluded packages
         run: |
+          rustup component add clippy rustfmt
+          rustup target add wasm32-unknown-unknown
+
+          rustup component add rust-src --toolchain nightly-2022-10-30-x86_64-unknown-linux-gnu
+          cargo install cargo-dylint dylint-link
+          cargo install cargo-contract --vers ^2.0 --force --locked
```

Comment:
I would have thought that in the old approach it wouldn't try to install it multiple times - it would _try_ but then realize it's already there.